OADP-5191: Fix Attributes in Support for OpenShift API for Data Prote… (#84648)
openshift/openshift-docs: modules/oadp-operator-supported.adoc

|===

|Version

> |OCP version

Reviewer: ```suggestion
|{product-title} version
```

Author: This is incorrect. this will not render in a table in the OCP enterprise

a|
27 Jun 2026

> EUS must be on OCP 4.16

Reviewer: ```suggestion
EUS must be on {product-title} 4.16
```

a|
27 Jun 2027

> EUS Term 2 must be on OCP 4.16

Reviewer: ```suggestion
EUS Term 2 must be on {product-title} 4.16
```

a|
31 Oct 2025

> EUS must be on OCP 4.14

Reviewer: ```suggestion
EUS must be on {product-title} 4.14
```

a|
31 Oct 2026

> EUS Term 2 must be on OCP 4.14

Reviewer: ```suggestion
EUS Term 2 must be on {product-title} 4.14
```
